src/marines/my_console.py

```diff
@@ -0,0 +1,18 @@
+def function1():
+    size = raw_input("please set the size of the array: ")
+    print(size)
+    myArray = []
+    for index in range(0, int(size)):
+        number = raw_input("please set the " + str(index + 1) + " value of the array: ")
+        myArray.append(number)
+    return myArray
+
+
+def function2(myArray):
+    print myArray
+
+def function3():
```

Comment:
what does function3 mean?

Reply:
it is for call function1 and function2.
Renamed by load_ and_print_array